### PR TITLE
Missing tracing duration on low risk detail screen (EXPOSUREAPP-5092)

### PR DESCRIPTION
**Pull request by LukasLechnerDev**
Issue: The active tracing history ("Exposure logging was active for x of the past 14 days.) was missing in the low-risk detail screen. It was only missing when there were encounters. 

Fix: I made it visible on the detail screen via editing the binding expression in the layout. I created a new binding adapter to simplify the expression.

To get a low-risk card with encounters do: Test Menu => Select LOW_RISK_DEFAULT => press "Calculate Risk Level".

Before: 
image

After:
image

### CODE REVIEW

**Review by harambasicluka — DISMISSED** [review]
Tested on a Pixel3, works as expected! Thanks :)

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/ui/ViewExtensions.kt [hunk 1]
```diff
@@ -4,6 +4,11 @@ import android.view.View
 import androidx.databinding.BindingAdapter
 import de.rki.coronawarnapp.util.recyclerview.ThrottledClickListener
 
+@BindingAdapter("visible")
+fun View.setVisible(visible: Boolean) {
```
Comment (ralfgehrer) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/ui/ViewExtensions.kt:8]: This is exactly  the same but negated as the `setGone` below

File: Corona-Warn-App/src/main/res/layout/tracing_content_low_view.xml [hunk 2]
```diff
@@ -74,7 +74,7 @@
 
         <androidx.constraintlayout.widget.ConstraintLayout
             android:id="@+id/row_saved_days"
-            gone="@{state.getRiskContactLast(context) != null}"
+            visible="@{state.getRiskContactLast(context) == null || state.inDetailsMode}"
```
Comment (ralfgehrer) [inline Corona-Warn-App/src/main/res/layout/tracing_content_low_view.xml:77]: Why not use `gone` instead. No point in maintaining both adapters IMHO

Comment (LukasLechnerDev) [reply Corona-Warn-App/src/main/res/layout/tracing_content_low_view.xml:77]: I implemented it with `gone` in the beginning and it looked like this: 
`gone="@{state.getRiskContactLast(context) != null &amp;&amp; !state.inDetailsMode}"`
which is really hard to understand (`&amp;&amp;` is the logical AND operator (&&) in data binding )

The version here with `visible` is much easier to understand. 

I don't see any "costs" of maintaining this additional adapter, but I see some value in simplifying the expression.

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/ui/ViewExtensions.kt (hunk 1 above)
Comment (LukasLechnerDev) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/ui/ViewExtensions.kt:8]: See other comment.

File: Corona-Warn-App/src/main/res/layout/tracing_content_low_view.xml (hunk 2 above)
Comment (ralfgehrer) [reply Corona-Warn-App/src/main/res/layout/tracing_content_low_view.xml:77]: How about moving the logic to the state object --> `gone=@{state.isGoneOnContentLowView(context)}` for  instance

Comment (LukasLechnerDev) [reply Corona-Warn-App/src/main/res/layout/tracing_content_low_view.xml]: Good idea. Moved the logic to the state object and wrote tests for it.

**Review by harambasicluka — APPROVED** [review]

**Review by ralfgehrer — APPROVED** [review]

**Pull request merged**